src/analytics/Properties.tsx

```diff
@@ -1553,6 +1553,23 @@ interface JumpstartEventsProperties {
   [JumpstartEvents.jumpstart_claim_loading_dismissed]: undefined
   [JumpstartEvents.jumpstart_claim_error_dismissed]: undefined
   [JumpstartEvents.jumpstart_claim_error_contact_support]: undefined
+  [JumpstartEvents.jumpstart_reclaim_press]: { networkId: NetworkId; transactionHash: string }
+  [JumpstartEvents.jumpstart_reclaim_start]: { networkId: NetworkId; transactionHash: string }
+  [JumpstartEvents.jumpstart_reclaim_failed]: undefined
```

Comment:
just noticed that only those two are missing `networkId` and `transactionHash`, while others have it. what do you think about sending this information with every signle event? i see one pro: we could easily track every single hash trough those events.

Reply:
Yeah, I didn't add it because I didn't have the information in the saga. But I guess I can pass it.